driver/ppm_fillers.c

```diff
@@ -5349,6 +5348,64 @@ int f_sys_fsconfig_x(struct event_filler_arguments *args)
 	return add_sentinel(args);
 }
 
+int f_sys_epoll_create_e(struct event_filler_arguments *args)
+{
+	unsigned long size;
+	int res;
+
+	/*
+	 * size
+	 */
+	syscall_get_arguments_deprecated(current, args->regs, 0, 1, &size);
+	res = val_to_ring(args, size, 0, false, 0);
+	if (unlikely(res != PPM_SUCCESS))
+		return res;
```

Comment:
```suggestion
	CHECK_RES(res);
```

same in the fillers below :)